### DISCUSSION TITLE
no Available QueryNode result

### DISCUSSION THREAD

**Discussion by Dingchao1003** [milvus-io/milvus#16217] [Q&A and General discussion]
no Available QueryNode result, filter reason [UnexpectedError] Assert "(field_meta.get_sizeof() == element.line_sizeof_)" at /go/src/github.com/milvus-io/milvus/internal/core/src/query/Plan.cpp:41

: id 432131054093729885

**Comment by russmuh** [comment]
Same here

**Reply by yhmo** [reply]
This error indicates the search expression is invalid. What is the expression you pass to search()?

**Comment by estebanpw** [comment]
Maybe a bit late but I had this error while using binary vectors on the Java Milvus SDK. The problem was that I was creating the individual `ByteBuffers` of size `vectorDim`. I fixed it by creating the buffers of size `vectorDim/8` (therefore storing each binary `0` or `1` as a bit in the byte, rather than one per byte)

Example creation of `ByteBuffers` in a list:

```java
        List<ByteBuffer> vectors = new ArrayList<>();
        int count = 1000*1000;
        int vectorDimension = 8;
        int byteCount = vectorDimension/8;
        Random r = new Random();
        for(int i=0; i<count; i++) {
            ByteBuffer bb = ByteBuffer.allocate(byteCount);
            for(int j=0; j<byteCount; j++)
                bb.put((byte) (r.nextInt() & 0x00000011));
            vectors.add(bb);
        }
```
